src/services/WolfDen.Application/Requests/Queries/Employees/GetEmployeeIDSignUp/GetEmployeeIDSignUPQueryHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using MediatR;
+using Microsoft.EntityFrameworkCore;
+using WolfDen.Application.DTOs.Employees;
+using WolfDen.Application.Requests.Queries.Employees.GetEmployeeIdSignUp;
+using WolfDen.Infrastructure.Data;
+
+namespace WolfDen.Application.Requests.Queries.Employees.GetEmployeeIDSignUp
+{
+    public class GetEmployeeIDSignUPQueryHandler(WolfDenContext context) : IRequestHandler<GetEmployeeIDSignUpQuery, EmployeeSignUpDto>
+    {
+        private readonly WolfDenContext _context = context;
+
+        public async Task<EmployeeSignUpDto> Handle(GetEmployeeIDSignUpQuery request, CancellationToken cancellationToken)
+        {
+            EmployeeSignUpDto result = new();
+            var employee = await _context.Employees.FirstOrDefaultAsync(x => x.EmployeeCode == request.EmployeeCode && x.RFId == request.RFId, cancellationToken);
+            if (employee == null)
+            {
+
+                result.Id = 0;
+                result.Message = string.Format("Employee Does not Exists!");
+                return result;
+            }
+            if (employee.Email != null)
+            {
+
+                result.Id = 0;
+                result.Message = string.Format("Employee Already Exists!");
+                return result;
+            }
+
+            result.Id = employee.Id;
+            result.Message = string.Format($"HI {employee.FirstName} {employee.LastName} Welcome To GeekyWolf");
+            return result;
+
```

Comment:
does not create  a message from Be directly, Be will return only the employee details like firstName, lastName 
and create the message in FE using this,
why here employee already exists logic ?,
if employee does not exists then throw the exception and you can get in the error part of subscribe in Fe and handle the message part there